keps/693-multikueue/README.md

```diff
@@ -231,11 +233,16 @@ has a valid configuration and at least one of its MultiKueueClusters is `Active`
 Will monitor the workloads in the management cluster and manage their MultiKueue specific
 AdmissionCheckStates.
 
-When distributing the workloads across clusters MultiKueue Workloads controller will first create
-the Kueue's internal Workload object. Only after the workload is admitted and other clusters
-are cleaned-up the real job will be created, to match the Workload. That gives the guarantee
-that the workload will not start in more than one cluster. The workload will
-get the annotation stating where it is actually running.
+When distributing workloads across clusters, the MultiKueue Workload Controller first nominates 
```

Comment:
To be fair, again, it feels like introducing more confusion than clarifying. First, the field `spec.multiKueueNominatedClusters` is introduced later. Second, the "the MultiKueue Workload Controller first nominates "  is not really true, the nomination is done by the dispatcher. It might be internal, or external. 

I think the previous version of the paragraph is actually capturing enough details. We can drill into details in the new  MultiKueue Dispatcher API section.